Chp12Lsn7/UnitTestProject/UnitTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Chp12Lsn7;
+using UserCommunication;
+
+namespace UnitTestProject
+{
+    [TestClass]
+    public class UnitTest
+    {
+        [TestMethod]
+        [ExpectedException(typeof(System.ArgumentException), "Invalid Input! Cannot enter a number less than or equal to zero.")]
+        public void SqrtTest()
+        {
+            int[] invalidNumbers = { 0, -2 };
+            int numberOfColumns = invalidNumbers.GetLength(0);
```

Comment:
When there is just one dimension, just do this:

    int numberOfColumns = invalidNumbers.Length;

Wheee!